### ISSUE TITLE
Drush 10 conflicts with drupal/migrate_tools: <= 5

### ISSUE THREAD

**Issue by andriyun** [OS2web/os2web_meetings#4]
Source https://packagist.org/packages/drush/drush

Drush 10 is a recent version of drush. Drupal 9 is working with drush 9 and higher.
Currently, builds on the project is failing because of this conflict

**Proposed solution:** 
update os2web_meetings to using drupal/migrate_tools => 5.0

**Comment by andriyun** [comment]
@stanbellcom Any thoughts about potential issues?

**Comment by stankut** [comment]
@andriyun 
i agree, the upgrading the the right way to go here. I don't think there will be major issues, but that needs to be tested.

We also need to keep in mind, that we have list of other modules, depending on drupal/migrate_tools. That is:
- os2web_borgerdk
- os2web_meetings_agman
- os2web_meetings_edoc
- os2web_meetings_sbsys

so update needs to be done for all of them all toghether.
